deploy/jenkins-ci/Jenkinsfile

```diff
@@ -112,17 +122,23 @@ node('kiali-build') {
         }
       }, ui: {
         stage('Build UI') {
-          sh "make -f ${uiMakefile} -C ${uiDir} ui-fix-version ui-build"
+          if ( !skipUi ) {
```

Comment:
I don't like doing negative checks all the time, maybe `!skipUi` should be something like `buildUi` on all those instances?

Reply:
ok, I changed it to be positive :)